packages/forma-36-react-components/src/components/IconButton/IconButton.tsx

```diff
@@ -5,7 +5,38 @@ import { Icon, IconProps } from '../Icon';
 import { TabFocusTrap } from '../TabFocusTrap';
 import styles from './IconButton.css';
 
-export interface IconButtonProps extends React.HTMLAttributes<HTMLElement> {
+type IconButtonAnchorProps =
+  | {
+      /**
+       * Used to make the decision of either rendering as a <a> or as a <button> tag
+       */
+      href?: undefined;
+      /**
+       * Used with href to define a relationship between a linked resource and the current document
+       */
+      rel?: never;
+      /**
+       * Used with href to specify target attribute value
+       */
+      target?: never;
+    }
+  | {
+      /**
+       * Used to make the decision of either rendering as a <a> or as a <button> tag
+       */
+      href: string;
+      /**
+       * Used with href to define a relationship between a linked resource and the current document
+       */
+      rel?: React.AnchorHTMLAttributes<HTMLAnchorElement>['rel'];
+      /**
+       * Used with href to specify target attribute value
+       */
+      target?: React.AnchorHTMLAttributes<HTMLAnchorElement>['target'];
+    };
+
```

Comment:
```suggestion
```

Reply:
@denkristoffer w8. I wanted to keep this change 😄